packages/cli/src/blue-harvest/api-keys.ts

```diff
@@ -0,0 +1,30 @@
+import { resolve } from 'path';
+import { homedir } from 'os';
+import { existsSync, readFileSync } from 'fs';
+
+const TOKEN_FILENAME = '.redocly-config.json';
+
+function isNotEmptyObject(obj: any) {
```

Comment:
Please use the one from `core/src/utils`.

Reply:
I cannot, should I export it from there?

Reply:
It is already being exported: https://github.com/Redocly/redocly-cli/blob/main/packages/core/src/utils.ts#L177